feat: add monitoring docs (#669)
GreptimeTeam/docs: docs/en/v0.4/user-guide/operations/monitor.md

> # Monitor GreptimeDB

Reviewer: Maybe can just the word **Monitoring** and it will be clean and short.

> To collect GreptimeDB metrics using Prometheus

Reviewer: ```suggestion
The GreptimeDB exposes the Prometheus metrics, and the users can also use [Prometheus](https://prometheus.io/) to collect the metrics.
```

> ## Promethues Configuration

Reviewer: Need to refactor the section title, for example:

```
## Docker

Use Docker to deploy Prometheus and GreptimeDB:

1. Prepare `prometheus.yml`

   ....

2. Start the Prometheus image

   ...

3. Start the GreptimeDB image

   ...

## Kubernetes

1. Install kube-prometheus-stack chart

2. Deploy the GreptimeDB cluster
```

> Write a Prometheus configuration file and save it as prometheus.yml:

Reviewer: ```suggestion
Write a Prometheus configuration file and save it as `prometheus.yml`:
```

```
./prometheus --config.file=prometheus.yml
```

Access Prometheus by entering `localhost:9090` in your web browser.

### Docker

Install GreptimeDB according to the [documentation](https://docs.greptime.com/getting-started/try-out-greptimedb#docker)

Run Prometheus:
```
docker run \
  -p 9090:9090 \
  -v $(pwd)/prometheus.yml:/etc/prometheus/prometheus.yml \
  prom/prometheus
```

### Kubernetes

Install [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
```
helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
helm repo update
helm install prometheus prometheus-community/kube-prometheus-stack
```

Use [gtctl](https://docs.greptime.com/user-guide/operations/gtctl) to install GreptimeDB cluster:
```
gtctl cluster create mycluster -n default \
  --set cluster.prometheusMonitor.enabled=true \
  --set cluster.prometheusMonitor.path="/metrics" \
  --set cluster.prometheusMonitor.port="http" \
  --set cluster.prometheusMonitor.interval="30s" \
  --set cluster.prometheusMonitor.honorLabels=true \
  --set cluster.prometheusMonitor.labelsSelector.release="prometheus"
```

## Metrics Detail

> To find update-to-date information, please check the output of `curl http://<host>:<port>/metrics`

Reviewer: ```suggestion
You can check the output of `curl http://<host>:<port>/metrics` by getting the latest metrics of GreptimeDB. We will add more documents of the metrics sooner.
```
